tests/test_netmiko_config.py

```diff
@@ -20,6 +20,9 @@ def test_enable_mode(net_connect, commands, expected_responses):
     Catch exception for devices that don't support enable
     """
     try:
+        if net_connect.device_type in ["nokia_srl"]:
+            assert pytest.skip()
```

Comment:
Shouldn't be necessary the `NoEnable` parent class should handle this properly such that the tests should work properly without any modifications.